spec/operators/catch-spec.ts

```diff
@@ -278,4 +279,83 @@ describe('Observable.prototype.catch', () => {
       done();
     });
   });
+
+  context('fromPromise', () => {
```

Comment:
I know this is a nitpick... but everywhere else I think we use `describe`. I didn't even know `context` is a thing. Is there a reason this is better in this case than `describe`?